app/src/main/java/com/lla/panicbutton/ui/fragments/SetupFragment.kt

```diff
@@ -23,10 +23,19 @@ class SetupFragment : Fragment(R.layout.fragment_setup) {
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
         super.onViewCreated(view, savedInstanceState)
 
+        if(onBoardingButton()) {
+            nextButton.visibility = View.GONE
+            doneButton2.visibility = View.VISIBLE
```

Comment:
why is it called doneButton2? what's doneButton? if there's already a done button, we can have better descriptive names to differentiate them.

Reply:
ohh is it that the 4 welcome fragments don't have separate scopes so you have to make sure they have different IDs?

Reply:
Yeah, when I tried using just the ID for a component, it becomes ambiguous. So, I gave them different ID numbers. I could give them more descriptive names then.